features/offline-audio-context.yml

```diff
@@ -0,0 +1,15 @@
+name: OfflineAudioContext
+description: The `OfflineAudioContext` API TODO.
```

Comment:
```suggestion
description: The `OfflineAudioContext` API represents an audio processing graph that writes to an `AudioBuffer` instead of an output device.
```